samples/web/content/apprtc/apprtc.py

```diff
@@ -361,17 +392,28 @@ def add_client_to_room(host, room_id, client_id, is_loopback):
       error = constants.RESPONSE_DUPLICATE_CLIENT
       break
 
+    if room.room_type != room_type:
+      error = constants.RESPONSE_INVALID_ROOM
```

Comment:
Can we add a warning log here? This particular error is should be at least extremely uncommon and invalid room isn't descriptive enough to debug what happened. (eg not allowed also returns invalid room)